### ISSUE TITLE
Add filter for Mesh column titles

### ISSUE THREAD

**Issue by strarsis** [linchpin/mesh#242]
It would be great if the title markup for Mesh columns can be filtered.

**Comment by aaronware** [comment]
Good idea. Might be better to make a mesh specific filter first so it can be explicitly targeted. We can also apply the_title filter as well. technically all blocks/columns are posts as well so they adhere to the_title filter. As you have seen you can also simply create a new template if you needed vs filtering. Or you could override the default template with your own as long as you use the same name. It accomplishes the same goal with out adding more and more filters
